src/__tests__/httpYacApi.test.ts

```diff
@@ -355,17 +355,18 @@ foo={{foo.foo}}
 POST http://localhost:8080/post?test={{foo.test}}
 
 foo={{foo.foo}}
-        `);
-      const result = await send({
+      `);
+
+      await send({
         httpFile,
         httpRegion: httpFile.httpRegions[1],
       });
-      expect(result).toBeTruthy();
+
+      const refRequests = await refEndpoints.getSeenRequests();
```

Comment:
I like the reason behind the change. The order of the tests corresponds to the order of execution.